backend_model/Comment_post.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require("mongoose")
+
+// schema for the comment
+const commentSchema = mongoose.Schema({
+    title: {
+        type: String,
+        required: false
+    },
+    content: {
+        type: String,
+        required: true
+    }
+    
+})
```

Comment:
I believe this schema is missing a `post` field; to know which post a comment is for.

Reply:
I added to the schema